src/app/prevalence/components/PrevalenceMainContent.tsx

```diff
@@ -17,6 +18,15 @@ const PrevalenceMainContent: React.FC<PrevalenceMainContentProps> = ({
         usePrevalenceFilters();
     const theme = useTheme();
     const { t } = useTranslation(["PrevalencePage"]);
+    const supportMail = useFetchSupportEmail(); // Use the fetched email
+
+    useEffect(() => {
+        if (showError && error && supportMail) {
+            window.location.href = `mailto:${supportMail}?subject=ZooNotify-Error-Report&body=${encodeURIComponent(
```

Comment:
Please don't set window.location.href.  React offers more elegant ways to manipulate the DOM.  We can talk about it on Monday